neptune/variables/atoms/float.py

```diff
@@ -24,8 +27,11 @@
 
 class Float(Atom):
 
-    def assign(self, value: float, wait: bool = False):
-        self._experiment._op_processor.enqueue_operation(AssignFloat(self._experiment._uuid, self._path, value), wait)
+    def assign(self, value: Union[float, int], wait: bool = False):
```

Comment:
Not worth it. Type verification is different. It would result in more complicated code actually (implementing `asign` in parent and calling `super` in childs).